plugins/catalog-import/src/components/ImportInfoCard/ImportInfoCard.tsx

```diff
@@ -77,10 +79,18 @@ export const ImportInfoCard = (props: ImportInfoCardProps) => {
         <>
           <Typography component="h4" variant="h6">
             Link to a repository{' '}
-            <Chip label="GitHub only" variant="outlined" size="small" />
+            <Chip
+              label="GitHub and Azure DevOps"
+              variant="outlined"
+              size="small"
+            />
+          </Typography>
+
+          <Typography variant="subtitle2" color="textSecondary" paragraph>
+            GitHub Example: <code>{exampleGitRepositoryUrl}</code>
           </Typography>
           <Typography variant="subtitle2" color="textSecondary" paragraph>
-            Example: <code>{exampleRepositoryUrl}</code>
+            Azure Example: <code>{exampleAzureRepositoryUrl}</code>
```

Comment:
```suggestion
            Azure DevOps Example: <code>{exampleAzureRepositoryUrl}</code>
```

Reply:
Should we also include an example for Azure DevOps Server?

`https://azure-devops.mycompany.com/team-collection-name/team-project-name/_git/repo-name`

Reply:
Another way to go would be to revert the changes here instead - this doesn't really scale; it's just examples anyway. If the original props were all we had, integrators could choose to put other examples in there that are perfect for their specific org. Not sure we should be extending the props here tbh

Reply:
Sure, I'm good with that! 🚀

Reply:
Thank you for coming back to me on this. Are we happy with the suggested change here ? Or do we want to revert the component's props to their original values  ? Happy to go with whatever you guys  feel is more appropriate

Sorry if I'm asking to many questions :)

Reply:
Thanks for reverting 👍